server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJamesServerTest.java

```diff
@@ -61,9 +62,28 @@ public void setup() throws Exception {
 
     protected abstract GuiceJamesServer<Id> createJamesServer();
 
+    protected abstract void clean();
+
     @After
     public void tearDown() throws Exception {
         server.stop();
+        clean();
+    }
+
+    @Test
+    public void hostnameShouldBeUsedAsDefaultDomain() throws Exception {
+        String expectedDefaultDomain = InetAddress.getLocalHost().getHostName();
```

Comment:
Doesn't this test depends on user environment ? Couldn't we set the default domain from the config instead ?